### PR TITLE
hirad/fix--Added condition to fix addIndicator undefined error

### PR DESCRIPTION
**Pull request by hirad-deriv**

### CODE REVIEW

File: src/indicatorBuilder.js [hunk 1]
```diff
@@ -127,7 +127,9 @@ const init = (chart_series, indicator) => {
          before_add_callback && before_add_callback();
 
          //Add indicator for the main series
-         chart_series[0].addIndicator(state.id, options);
+         if (chart_series !== "undefined") {
+            chart_series[0].addIndicator(state.id, options);
+         }
```
Comment (matin-deriv) [inline src/indicatorBuilder.js:132]: Can we making it simpler? 
```suggestion
            chart_series[0]?.addIndicator(state.id, options);
```

Comment (hirad-deriv) [reply src/indicatorBuilder.js:132]: I was not sure if optional chaining works as expected in older projects. I made the code simpler as requested.

Comment (matin-deriv) [reply src/indicatorBuilder.js:132]: Yes, you are right! my bad, optional chaining is not working as expected in older project!

**Review by matin-deriv — APPROVED** [review]

File: src/indicatorBuilder.js [hunk 2]
```diff
@@ -127,7 +127,9 @@ const init = (chart_series, indicator) => {
          before_add_callback && before_add_callback();
 
          //Add indicator for the main series
-         chart_series[0].addIndicator(state.id, options);
```
Comment (mahdiyeh-deriv) [inline src/indicatorBuilder.js:130]: can we use if (chart_series && chart_series[0]) instead?

Comment (hirad-deriv) [reply src/indicatorBuilder.js:130]: It would definitely work but isn't it better to just stick to the structure that the project has ?

**Pull request merged**